examples/seq2seq/test_fsmt_bleu_score.py

```diff
@@ -0,0 +1,184 @@
+            """Ein Labour-Abgeordneter sagte, dass seine Gruppe "sich mit Twp und Pwp reimt".""",
+            """Hinweis für den Leser: „twp“ im Walisischen bedeutet „bescheuert“ und „pwp“ bedeutet „Kacke“.""",
+            """Ein Versammlungsmitglied von Plaid Cymru sagte, die Gruppe als Ganzes sei "nicht glücklich" und hat Alternativen vorgeschlagen.""",
+            """Ein walisischer Konservativer sagte, seine Gruppe wäre „offen“ für eine Namensänderung, wies aber darauf hin, dass es von „MWP“ (Mitglied des Walisischen Parlaments) nur ein kurzer verbaler Sprung zu „Muppet“ ist.""",
+        ],
+    },
+    "de-en": {
+        "src": [
+            """Schöne Münchnerin 2018: Schöne Münchnerin 2018 in Hvar: Neun Dates""",
+            """Von az, aktualisiert am 04.05.2018 um 11:11""",
+            """Ja, sie will...""",
+            """\"Schöne Münchnerin" 2018 werden!""",
+            """Am Nachmittag wartet erneut eine Überraschung auf unsere Kandidatinnen: sie werden das romantische Candlelight-Shooting vor der MY SOLARIS nicht alleine bestreiten, sondern an der Seite von Male-Model Fabian!""",
+            """Hvar - Flirten, kokettieren, verführen - keine einfachen Aufgaben für unsere Mädchen.""",
+            """Insbesondere dann, wenn in Deutschland ein Freund wartet.""",
+            """Dennoch liefern die neun "Schöne Münchnerin"-Kandidatinnen beim Shooting mit People-Fotograf Tuan ab und trotzen Wind, Gischt und Regen wie echte Profis.""",
+        ],
+        "tgt": [
+            """The Beauty of Munich 2018: the Beauty of Munich 2018 in Hvar: Nine dates""",
+            """From A-Z, updated on 04/05/2018 at 11:11""",
+            """Yes, she wants to...""",
+            """to become "The Beauty of Munich" in 2018!""",
+            """In the afternoon there is another surprise waiting for our contestants: they will be competing for the romantic candlelight photo shoot at MY SOLARIS not alone, but together with a male-model Fabian!""",
+            """Hvar with its flirting, coquetting, and seduction is not an easy task for our girls.""",
+            """Especially when there is a boyfriend waiting in Germany.""",
+            """Despite dealing with wind, sprays and rain, the nine contestants of "The Beauty of Munich" behaved like real professionals at the photo shoot with People-photographer Tuan.""",
+        ],
+    },
+}
+
+
+@require_torch
+class ModelTester(unittest.TestCase):
+    def get_tokenizer(self, mname):
+        return FSMTTokenizer.from_pretrained(mname)
+
+    def get_model(self, mname):
+        model = FSMTForConditionalGeneration.from_pretrained(mname).to(torch_device)
+        if torch_device == "cuda":
+            model.half()
+        return model
+
+    @parameterized.expand(
+        [
+            ["en-ru", 28.21],
+            ["ru-en", 23.49],
+            ["en-de", 22.11],
+            ["de-en", 29.31],
+        ]
+    )
+    @slow
+    def test_bleu_scores(self, pair, min_bleu_score):
```

Comment:
<3 
how long does it take on CPU?

Reply:
Are you asking whether this should be not 'slow'? the problem is the downloading of the model

I think it takes about the same time, but I can measure if it's important.

Reply:
```
time RUN_SLOW=1 pytest -sv --disable-warnings examples/seq2seq/test_fsmt_bleu_score.py::ModelTester::test_bleu_scores_0_en_ru
```
```
real    0m14.856s
user    0m35.047s
sys     0m2.110s
```

with USE_CUDA=1
```
real    0m13.690s
user    0m11.270s
sys     0m2.083s
```

Reply:
I just wanted to know whether it should only run on CUDA, but I think it should run on both.